packages/office-ui-fabric-react/src/common/DocPage.types.ts

```diff
@@ -135,4 +135,39 @@ export interface IDocPageProps {
     title: string;
     section: JSX.Element;
   }[];
+
+  jsonDocs?: IPageJson;
```

Comment:
Are these interfaces used anywhere in OUFR? If not, they should probably be removed. If so, ApiReferencesTableSet.types should reference these interfaces instead of duplicating them.